tests/system/action/user/test_update.py

```diff
@@ -128,6 +146,391 @@ def test_update_temporary_user_error(self) -> None:
         response = self.request("user.update", {"id": 5, "username": "username5"})
         self.assert_status_code(response, 400)
         self.assertIn(
-            "Please use Action user.update_temporary for temporary user",
+            "User 5 in payload may not be a temporary user.",
+            response.json["message"],
+        )
+
+    def test_update_permission_nothing(self) -> None:
+        self.permission_setup()
+        response = self.request(
+            "user.update",
+            {
+                "id": 111,
+                "username": "username_Neu",
+                "vote_weight_$": {1: "1.000000"},
+                "group_$_ids": {1: [1]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
```

Comment:
same here: I would expect it to fail in the first place of not having the right OML.

Reply:
There is no fixed sequence of showing the missing rights, because the field dict for checking has none.
The alternative is always mentioned, if it is the first in a rightlist of a field and because it's the only case, it's always the missing committee right.